docs-kits/kits/Digital Twin Kit/Software Development View/page_interaction-patterns.md

```diff
@@ -0,0 +1,134 @@
+---
+id: dt-kit-interaction-patterns
+title: Interaction Patterns
+description: 'Digital Twin KIT - Interaction Patterns'
+sidebar_position: 2
+---
+
+## Interaction Patterns for Distributed Digital Twins
+
+The Catena-X standard CX-0002 (Digital Twins in Catena-X) defines a subset of the
+
+### 1. Fetching a supplier's Twin
+
+| Scenario                                                                    | Participants                                           | Assumptions                                                                 | Links to Use-Cases                                                                               |
+|-----------------------------------------------------------------------------|--------------------------------------------------------|-----------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|
+| The supplier has data required by the OEM. For instance a Bill of Material. | - Data Provider (Supplier) <br/> - Data Consumer (OEM) | 1. Consumer knows Supplier's BPN <br/> 2. Consumer knows an id of the asset | - Industry Core (coming soon) <br/> - [Product Carbon Footprint Kit](../../PCF%20Exchange%20Kit) |
```

Comment:
The BOM is not a good example. The OEM has a BOM and this is how the OEM knows whom to ask for data. Example data would be PCF.